config.toml

```diff
@@ -2,20 +2,21 @@ baseURL = "https://pulumi.io/"
 languageCode = "en-us"
 title = "Pulumi"
 
-disableKinds = ["category", "RSS"]
+disableKinds = ["category", "taxonomyTerm"]
```

Comment:
Removing `taxonomyTerm` as well here just to silence a build warning: 

```
WARN 2019/06/14 10:48:59 found no layout file for "HTML" for "taxonomyTerm"
```

I don't think we _need_ this, but I'm not yet an expert on Hugo taxonomies, so given our author and tag pages are rendering properly, it seems fine to suppress it. @justinvp may know better, though.